perception_dataset/rosbag2/autoware_msgs.py

```diff
@@ -1,13 +1,23 @@
 from typing import Any, Dict, List, Optional, Union
 import uuid
 
-from autoware_auto_perception_msgs.msg import (
-    DetectedObject,
-    DetectedObjects,
-    ObjectClassification,
-    TrackedObject,
-    TrackedObjects,
-)
+try:
+    from autoware_auto_perception_msgs.msg import (
+        DetectedObject,
+        DetectedObjects,
+        ObjectClassification,
+        TrackedObject,
+        TrackedObjects,
+    )
+except ModuleNotFoundError:
```

Comment:
[NITS] How about display the error message?

Reply:
Thanks. Fixed: https://github.com/tier4/tier4_perception_dataset/pull/104/commits/c22614c1977f4a991e61f36a7aaeda9f19a6d280